sql/core/src/main/scala/org/apache/spark/sql/execution/Window.scala

```diff
@@ -920,7 +931,7 @@ private[execution] object AggregateProcessor {
     // Create the projections.
     val initialProjection = newMutableProjection(
       initialValues,
-      Seq(SizeBasedWindowFunction.n))()
+      Seq(partitionSize))()
```

Comment:
OK I see. I think I am confused about the reason that the inputSchema of this mutable projection only contains a single column `partitionSize`. Does initialValues already have all other needed BoundReferences?